orm/query.go

```diff
@@ -570,6 +570,11 @@ func (q *Query) WhereIn(where string, slice interface{}) *Query {
 	return q.Where(where, types.In(slice))
 }
 
+// WhereInOr is a shortcut for WhereOr and pg.In.
+func (q *Query) WhereInOr(where string, slice interface{}) *Query {
+	return q.WhereOr(where, types.In(slice))
+}
```

Comment:
Would you be able to add a test proving the query is built how you intend it to be built with this function. Like in `orm/select_nil_test.go`